src/js/utils/specs/string-utils-spec.js

```diff
@@ -0,0 +1,58 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../string-utils.js'
+    ],
+    function(SpecHelpers, StringUtils) {
+        'use strict';
+
+        describe('StringUtils', function() {
+            describe('interpolateEscape', function() {
+                it('can interpolate a string with no parameters provided', function() {
+                    expect(StringUtils.interpolateEscape('Hello, world').toString()).toBe(
```

Comment:
I see.  Then maybe below you can change the comment to "...with empty parameters".